src/Sulu/Bundle/ContactBundle/Infrastructure/Sulu/Search/ContactReindexProvider.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContactBundle\Infrastructure\Sulu\Search;
+
+use CmsIg\Seal\Reindex\ReindexConfig;
+use CmsIg\Seal\Reindex\ReindexProviderInterface;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\EntityRepository;
+use Sulu\Bundle\ContactBundle\Entity\ContactInterface;
+
+/**
+ * @phpstan-type Contact array{
+ *     id: int,
+ *     changed: \DateTimeImmutable,
+ *     created: \DateTimeImmutable,
+ *     firstName: string,
+ *     lastName: string,
+ * }
+ */
+class ContactReindexProvider implements ReindexProviderInterface
+{
+    /**
+     * @var EntityRepository<ContactInterface>
+     */
+    protected EntityRepository $contactRepository;
+
+    public function __construct(
+        EntityManagerInterface $entityManager,
+    ) {
+        $repository = $entityManager->getRepository(ContactInterface::class);
+
+        $this->contactRepository = $repository;
+    }
+
+    public function total(): ?int
+    {
+        return null;
```

Comment:
We can implement this:

```suggestion
        return $this->contactRepository->count();
```